docs/platform/index.mdx

```diff
@@ -0,0 +1,33 @@
+---
+displayed_sidebar: platformSidebar
+sidebar_position: 1
+sidebar_label: Overview
+title: Platform
+description: Learn about the Momento Platform and the capabilities it provides to your architecture
+pagination_next: null
+hide_title: true
+keywords:
+  - momento
+  - cache
+  - caching
+  - topics
+  - eda
+  - event-driven architecture
+  - redis
+  - serverless
+---
+
+# The Momento Platform
+
+Ship better products, faster with a platform designed for mission-critical workloads. Momento makes infrastructure easy at enterprise scale with [caching](/cache), [unrivaled event messaging](/topics), and soon, *storage*!
+
+import { CardGrid } from '@site/src/components/CardGrid';
+import { LinkCard } from '@site/src/components/LinkCard';
+
+<CardGrid>
+  <LinkCard alignText='center' link="/cache/getting-started" title="Get started" description="Quickly get up and running with Momento Cache." />
+  <LinkCard alignText='center' link="https://console.gomomento.com/topics" title="Send a message" description="Try out Topics for free directly in your browser" />
+  <LinkCard alignText='center' link="/platform/regions" title="Availability" description="View the available regions around the globe" />
+  <LinkCard alignText='center' link="/platform/sdks" title="Supported languages" description="Find your favorite language in our list of SDKs" />
```

Comment:
super nit, not a blocker, but when i clicked on platform i was hoping "SDKs" would be more visible on the next page than it ended up being. I eventually found it in the small text (and left nav). I think I'd prefer that the title of that card just say "SDKs" so it was more visible, but YMMV